### DISCUSSION TITLE
Intégration Moodle ?

### DISCUSSION THREAD

**Discussion by ChristianSimonSU** [seb-cogez/QCMCam#3] [Ideas]
Bonjour,

Est-ce que certains des utilisateurs ont déjà tenté une intégration à Moodle ?
(création de la liste de participants depuis les inscrits à un cours, et réciproquement importation dans Moodle des résultats)

**Comment by seb-cogez** [comment]
Pas à ma connaissance. Et ce n'est pas prévu pour car les réponses ne sont pas données de manière informatique et individualisées.

**Reply by ChristianSimonSU** [reply]
Merci !
